Bottom: 1406475ec3d5c2d8675bc2368637a56aea08d8eb
Top:    174f4fa3e9b417127e1c74e0900ccedbcbdc3d4b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:00 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 65edb50..2bca1f3 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
